website/content/docs/builders/file.mdx

```diff
@@ -1,23 +1,18 @@
 ---
 description: |
-  The file Packer builder is not really a builder, it just creates an artifact
-  from a file. It can be used to debug post-processors without incurring high
-  wait times.
-page_title: File - Builders
+  The `file` builder creates an artifact from a file. Use `file` to debug post-processors without incurring long wait times.
```

Comment:
```suggestion
  The `file` builder creates an artifact from a file. Use the `file` builder to debug post-processors without incurring long wait times.
```

I'm fine keeping it how you have it, especially if it's for character count reasons, but wanted to put this suggestion out there.